web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -440,13 +492,26 @@ export class BarChartComponent implements AfterViewInit {
               'height',
               (d: Item) => this.yScaleFocus(d.durationHours) -
                   this.yScaleFocus(maxFocusDuration))
-          .attr('fill', BarChartComponent.COLOR_WHITE_TRANS);
-      // TODO: Call mouseover and mouseleave functions here.
+          .attr('fill', BarChartComponent.COLOR_WHITE_TRANS)
+          .on('mouseover',
+              (d: Item, i: number) => {
+                d3.select(solidBars.nodes()[i]).attr('fill-opacity', 0.7);
```

Comment:
The stroke around the bars is not subject to this opacity. Is that intentional? One option I would suggest trying is removing the stroke altogether.